community/cython/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=cython
+pkgver=0.29.6
+pkgrel=0
+pkgdesc="Cython is an optimising static compiler for both the Python & the extended Cython programming languages."
+url="https://cython.org"
+arch="all"
+license="Apache-2.0"
+_py2_deps="python2-dev"
+_py3_deps="python3-dev"
```

Comment:
Why did you define these two vars? I see them only used 1 time.

Reply:
I used the "Python package policies" and copy-pasted the example. I have removed these two variables.